azure-pipelines/util/build_appimage.sh

```diff
@@ -11,8 +11,13 @@ if [ -z "$APPDIR" ]; then
     exit 1
 fi
 
-# linuxdeploy location
-LINUXDEPLOY=${LINUXDEPLOY:-"linuxdeploy.AppImage"}
+# get linuxdeploy and linuxdeploy-plugin-gtk
+# Currently broken: wget -c "https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-gtk/master/linuxdeploy-plugin-gtk.sh", see https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/issues/7
+wget -c "https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-gtk/c008df652946408f357f502bab67bfcf6f303b4e/linuxdeploy-plugin-gtk.sh"
+wget -c "https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage"
```

Comment:
The version of linuxdeploy should be set to a specific commit if possible so we don't get random breakages.

Reply:
That does not look possible. There is only a continuous release build, see https://github.com/linuxdeploy/linuxdeploy/releases

Reply:
I think it is recommended to have some backup of this tool just in case, because it may break one day.  
Example: https://github.com/linuxdeploy/linuxdeploy/issues/143.  
You can use older binaries from link in that issue: https://artifacts.assassinate-you.net//linuxdeploy/

Reply:
Please note that the artifacts storage is making no guarantees about its stability. For instance, I've recently changed from (the annoying) Artifactory to a plain HTTP server. If you are afraid that linuxdeploy is going to go down, you need to host your own binaries.